app/views/admin/home.html.haml

```diff
@@ -0,0 +1,2 @@
+- content_for(:page_header) do
+  = "Dashboard"
```

Comment:
One good practice is to always use `i18n` when displaying text. You never know when we might need to support other languages in the future :)